src/controllers/reminder.ts

```diff
@@ -11,18 +11,39 @@ export class ReminderController {
           .then(() => res.sendStatus(204))
           .finally(() => next());
       })
-      .put('/:userID/:lastAssignment', (req, res, next) =>{
+      .put('/:userID/:lastAssignment', (req, res, next) => {
         db.collection(Collections.users)
-        .doc(req.params.userID)
-        .update({'canvas.lastAssignment': req.params.lastAssignment})
-        .then(() => res.sendStatus(204))
-        .finally(()=> next());
+          .doc(req.params.userID)
+          .update({ 'canvas.lastAssignment': req.params.lastAssignment })
+          .then(() => res.sendStatus(204))
+          .finally(() => next());
       })
       .delete('/', (req, res, next) => {
         db.collection(Collections.reminders)
           .doc((req.body.id)).delete()
           .then(() => res.sendStatus(204))
           .finally(() => next());
+      })
+      .get('/offset/:id', async (req, res, next) => {
```

Comment:
Offset is a non-descriptive name. Suggest timezone or similar to what fits and describes.

Reply:
first used offset changed after forgot to change 👍